env-hooks/1.ros_etc_dir.sh.em

```diff
@@ -0,0 +1,12 @@
+# generated from ros_environment/env-hooks/1.ros_etc_dir.sh.em
+
+@[if DEVELSPACE]@
+# env variable in develspace
+export ROS_ETC_DIR="@(CATKIN_DEVEL_PREFIX)/@(CATKIN_GLOBAL_ETC_DESTINATION)/ros"
+@[else]@
+# env variable in installspace
+if [ -z "$CATKIN_ENV_HOOK_WORKSPACE" ]; then
```

Comment:
Is the use of `CATKIN_ENV_HOOK_WORKSPACE` not supported in the windows case?

Reply:
This is the current logic from `roslib`.